tests/bears/DebugBearsTest.py

```diff
@@ -1,9 +1,22 @@
+import multiprocessing
 import unittest
 import sys
 
 from io import StringIO
 
-from coalib.bears.Bear import Debugger, debug_run
+from coalib.bears.Bear import Bear, Debugger, debug_run
+from coalib.bears.LocalBear import LocalBear
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
```

Comment:
Line is longer than allowed. (86 > 80)

*Origin: LineLengthBear, Section: `linelength`.*